### PR TITLE
Refresh proxy locale when updating

### PR DESCRIPTION
**Pull request by m4dz**
This PR fix a locale loading bug that occurs when update locale settings in home: the new setting wasn't applied since the proxy is restarted. This PR fix it by forcing locale refresh into the proxy when accessing a frontend page (register / index / password-reset).

### CODE REVIEW

File: server/controllers/authentication.coffee [hunk 1]
```diff
@@ -12,18 +12,33 @@ otpManager   = require '../lib/2fa_manager'
 
 
 getEnv = (callback) ->
-    User.getUsername (err, username) ->
-        return callback err if err
+    async.parallel(
+        env: (callback) ->
```
Comment (nono) [inline server/controllers/authentication.coffee]: @m4dz you should rename `callback` in `done`, to avoid confusion with the callback from `getEnv`

**Comment by nono** [comment]
@m4dz the build is red, with many `Cannot find module '../locales/en_US.UTF-8'` errors

**Comment by m4dz** [comment]
Hmmm, seems to be a glitch in request-json when it transmits accepted locales. I investigate…

**Comment by frankrousseau** [comment]
@m4dz What is the status of this PR?

**Comment by m4dz** [comment]
Unfortunately, test always fail on Travis, and liiterally crash locally (and I couldn't find enough time to fix it). I'll try as soon as possible to close this one, I just need to fix this damned headless test and everything'll bo ok :smiley:.

**Comment by frankrousseau** [comment]
ping @m4dz

**Comment by m4dz** [comment]
I'm on it

**Comment by nono** [comment]
@m4dz any news?

**Comment by m4dz** [comment]
Always the same shit with local tests :disappointed:. I'll try to fix it next monday!

**Comment by frankrousseau** [comment]
This PR is opened from may 27. I think it's time to close it. The addition is interesting but it looks like noone has time to take care of it. What do you thing @m4dz?

**Comment by m4dz** [comment]
@frankrousseau I should agree with you: I can't find any sufficient time to properly fix tests env. Let's close it, and I'll re-open later when I can finally fix this issue :disappointed:.

**Pull request closed without merging**